shared/chat/routes.native.js

```diff
@@ -31,7 +31,7 @@ const conversationRoute = new RouteDefNode({
     },
     messageAction: {
       component: MessagePopup,
-      tags: {layerOnTop: true},
+      tags: {keepKeyboard: true, layerOnTop: true},
```

Comment:
Could you please make this name more specific about timing, something like "keepKeyboardOnLeave"?